treenav/apps.py

```diff
@@ -16,3 +16,24 @@ def ready(self):
         # now, we need to reconnect the post_save handler because unittests run
         # immediately after running migrate, in the same process
         post_migrate.connect(connect_post_save_handler)
+
+
+def setup_test_app(package, label=None):
+    """
+    Setup a Django test app for the provided package to allow test models
+    tables to be created if the containing app has migrations.
+
+    This function should be called from app.tests __init__ module and pass
+    along __package__.
```

Comment:
Could you add a link to the django ticket (that you referenced in your PR description) here?